### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by canad1an**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -2954,5 +2954,14 @@
     "telegram": "StakeWithPride",
     "operatorName": "",
     "github": ""
-  }
+  },
+  {
+ "ticker": "SOLRP",
+ "poolId": "9728b10a926c048af938e5c52053319db5be921e8b698842c3afd3cc",
+ "social": {
+   "twitter": "SOLRP_StakePool"
```
Comment (Bez625) [inline registry.json:2962]: Please can you add the other social keys, e.g. reddit, website etc? They need to be present, just leave them blank if you do not have those socials.

File: registry.json [hunk 2]
```diff
@@ -2954,5 +2954,14 @@
...
+ },
+ "telegram": "solrp_ada_pool",
+ "github": "canad1an"
```
Comment (Bez625) [inline registry.json:2965]: Also please add in operatorName. This must be either your real first name, real full name or just blank.

File: registry.json (hunk 1 above)
Comment (Bez625) [inline registry.json:2962]: Please can we see some activity on the twitter account?

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (canad1an) [reply registry.json]: Added.

Comment (canad1an) [reply registry.json]: I just opened the twitter actually, I will start posting some promotional info about our pool soon. Thanks!

File: registry.json (hunk 2 above)
Comment (canad1an) [reply registry.json:2971]: Done

**Pull request merged**